client/response.go

```diff
@@ -0,0 +1,104 @@
+package client
+
+import (
+	"time"
+)
+
+type BlockResponse struct {
```

Comment:
We can use [this website](https://transform.tools/) if we want JSON to typed structs translation.

Reply:
Yeah, I think I used @mholt's [https://github.com/mholt/json-to-go](translator).